matlab/Ice/context/client.m

```diff
@@ -16,11 +16,10 @@ function client(args)
     % Set the Ice.ImplicitContext property to 'Shared' before calling Ice.initialize.
     % This is only necessary for the implicit context API (see below).
     initData = Ice.InitializationData();
-    [props, ~] = Ice.createProperties(args);
-    initData.properties_ = props;
+    initData.properties_ = Ice.createProperties(args);
```

Comment:
Simplified this code. `createProperties(args)` and all initialize with args return 2 parameters, but we don't care about the second one.